swagger/swagger-generator/generator-springmvc/src/main/java/io/servicecomb/swagger/generator/springmvc/processor/annotation/RequestPartAnnotationProcessor.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2017 Huawei Technologies Co., Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.servicecomb.swagger.generator.springmvc.processor.annotation;
+
+import org.springframework.web.bind.annotation.RequestPart;
+
+import io.servicecomb.swagger.generator.core.processor.parameter.AbstractParameterProcessor;
+import io.swagger.models.parameters.FormParameter;
+import io.swagger.models.properties.FileProperty;
+
+public class RequestPartAnnotationProcessor extends AbstractParameterProcessor<FormParameter> {
+  @Override
+  protected FormParameter createParameter() {
+    FormParameter parameter = new FormParameter();
+    parameter.setType(new FileProperty().getType());
```

Comment:
type will be set in io.swagger.models.parameters.AbstractSerializableParameter.setProperty(Property)
no need to invoke setType here.